.github/CODEOWNERS

```diff
@@ -7,18 +7,18 @@
 
```

Comment:
You missed removing one reference to `jasonodoom` here:
https://github.com/cisagov/ipsec-tunnel-tf-module/blob/1b63633056b6efdb3d974a4b3f9b25b9df4e2d48/.github/CODEOWNERS#L6

Reply:
Please see commit b23f465.